lib/core/layout/components/header/header.component.html

```diff
@@ -0,0 +1,11 @@
+<mat-toolbar color="{{color}}">
+    <button class="mat-icon-button adf-menu-icon" mat-icon-button="" (click)="toggleMenu()">
```

Comment:
What if I don't want to have a button? ( Think about APW, when we show the apps we don't have any sidenav)

Reply:
I agree, maybe we have `showSidenavToggle` property or something like that?